HouseRules_Essentials/Rules/CardAdditionOverriddenRule.cs

```diff
@@ -67,10 +67,11 @@ private static void Patch(Harmony harmony)
                 return;
             }
 
+            _isPotionStand = false;
             Interactable whatIsit = gameContext.pieceAndTurnController.GetInteractableAtPosition(targetTile);
             if (whatIsit.type == Interactable.Type.PotionStand)
             {
-                _isChest = false;
+                _isPotionStand = true;
```

Comment:
QuickQuestion:  Unless `_isPotionStand` is reset when a card is granted, does this mean that this following scenario might happen?

- Player visits potion stand.  `_isPotionStand` set to `true`.
- Player gets an unaltered card.
- Player tops off their card energy.
- Since `_isPotionStand` is still `true`, player will get another unaltered card.
-

Reply:
Didn't notice that, thanks! Should be fixed with my latest commit.

Reply:
Whoops, not ready yet. Forgot it will cycle through when multiplayer. Will fix that here in a few and update.